templates/taxi/driver_detail.html

```diff
@@ -23,4 +25,6 @@ <h4>Cars</h4>
       <p>No cars!</p>
     {% endfor %}
   </div>
+  <a href="{% url 'taxi:driver-delete' pk=driver.id %}" class="btn btn-danger">Delete</a>
```

Comment:
According to the checklist, you should add a `Cancel` button. This button should lead to the previous page the user was on.